packages/components/atoms/f-filter-pill/src/components/FilterPill.vue

```diff
@@ -115,17 +117,18 @@ $filter-pill-checkbox-width: 16px;
 }
 
 .c-filterPill {
-    outline: 1px solid $color-grey-30;
+    box-shadow: 0 0 0 1px $color-grey-30;
```

Comment:
the `outline` was changed to `box-shadow` to get rounded corners on the outline in Safari. I'm super happy with this solution, but other solutions are more ugly. 😬

Reply:
Nice! I might need to update the imageTile to do the same ✨

Reply:
I know this wasn't added in this PR but I missed it previously but `$color-grey-30` is a global color variables which we should avoid using directly. So potentially could updated this to `$color-border-default` instead (https://unpkg.com/browse/@justeat/pie-design-tokens@1.4.0/dist/jet.scss)

Reply:
thanks! Will fix it in this PR